### PR TITLE
Release 0.55.1

### PR DESCRIPTION
**Pull request by pongad**

### CODE REVIEW

File: google-api-grpc/pom.xml [hunk 1]
```diff
@@ -134,407 +134,407 @@
...
         <groupId>com.google.api.grpc</groupId>
         <artifactId>grpc-google-cloud-vision-v1p2beta1</artifactId>
-        <version>1.19.1-SNAPSHOT</version><!-- {x-version-update:grpc-google-cloud-vision-v1p2beta1:current} -->
+        <version>1.19.1</version><!-- {x-version-update:grpc-google-cloud-vision-v1p2beta1:current} -->
       </dependency>
       <dependency>
         <groupId>com.google.api.grpc</groupId>
         <artifactId>proto-google-cloud-vision-v1p3beta1</artifactId>
-        <version>0.19.1-SNAPSHOT</version><!-- {x-version-update:proto-google-cloud-vision-v1p3beta1:current} -->
+        <version>0.20.1</version><!-- {x-version-update:proto-google-cloud-vision-v1p3beta1:current} -->
```
Comment (nnegrey) [inline google-api-grpc/pom.xml]: Is this one right?

File: google-cloud-bom/pom.xml [hunk 2]
```diff
@@ -223,792 +223,792 @@
...
         <groupId>com.google.api.grpc</groupId>
         <artifactId>grpc-google-cloud-vision-v1p2beta1</artifactId>
-        <version>1.19.1-SNAPSHOT</version><!-- {x-version-update:grpc-google-cloud-vision-v1p2beta1:current} -->
+        <version>1.19.1</version><!-- {x-version-update:grpc-google-cloud-vision-v1p2beta1:current} -->
       </dependency>
       <dependency>
         <groupId>com.google.api.grpc</groupId>
         <artifactId>proto-google-cloud-vision-v1p3beta1</artifactId>
-        <version>0.19.1-SNAPSHOT</version><!-- {x-version-update:proto-google-cloud-vision-v1p3beta1:current} -->
+        <version>0.20.1</version><!-- {x-version-update:proto-google-cloud-vision-v1p3beta1:current} -->
```
Comment (nnegrey) [inline google-cloud-bom/pom.xml]: Checking?

**Comment by pongad** [comment]
I think those 3 places were the reason it was wrong before. I'm 95% sure this will make it right now.

**Comment by pongad** [comment]
@hzyi-google is here. Let's wait for him to take a look to make sure.

**Comment by nnegrey** [comment]
Sounds good on waiting

**Comment by hzyi-google** [comment]
I can add another 4.9% to the sureness. LGTM

**Comment by pongad** [comment]
CI should finish soon. That should add another 0.05%.

**Pull request merged**